ts/components/ContextualHelpModal.tsx

```diff
@@ -43,101 +45,110 @@ const styles = StyleSheet.create({
 
 /**
  * A modal to show the contextual help reelated to a screen.
- * the contextual help is characterized by:
+ * The contextual help is characterized by:
  * - a title
  * - a textual or a component containing the screen description
  * - [optional] if on SPID authentication once the user selected an idp, content to link the support desk of the selected identity provider
  * - a list of questions and aswers. They are selected by the component depending on the cathegories passed to the component
+ *
+ * Optionally, the title and the content are injected from the content presented in the related clinet response.
  */
-export class ContextualHelpModal extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      content: null
-    };
-  }
+const ContextualHelpModal: React.FunctionComponent<Props> = (props: Props) => {
+  const [content, setContent] = React.useState<React.ReactNode | null>(null);
```

Comment:
```suggestion
  const [content, setContent] = React.useState<React.ReactNode>(null);
```
`React.ReactNode` could be null in its definition

Reply:
updated within https://github.com/pagopa/io-app/pull/1898/commits/732737daf8e67773abbeb723ee18a72f1004a450